vector/src/androidTest/java/im/vector/app/ui/UiAllScreensSanityTest.kt

```diff
@@ -196,6 +196,8 @@ class UiAllScreensSanityTest {
         pressBack()
         clickMenu(R.id.video_call)
         pressBack()
+        clickMenu(R.id.search)
+        pressBack()
```

Comment:
Thanks. Did you run the test?

Reply:
I couldn't reproduce the first crash. According to several rageshakes this is not the root cause.

Yes, I run the test and passed (it was fun to watch :)